config/wikis/huwiki.yaml

```diff
@@ -15,6 +15,9 @@ trusted_groups:
     - checkuser
     - abusefilter
     - bureaucrat
+    - editor
+    - templateeditor
+    - interface-editor
```

Comment:
Adding these now will cause some warnings, but no big issues.  Essentially, we'll have labels for some edits where "needs_review" is false with "reason" == "trusted_group"